docs/install.md

```diff
@@ -43,10 +43,8 @@ PDO in SGX HW mode, the PDO component has to run in an SGX-enabled
 environment. Below installation and configuration instructions will make
 sure that the host and the docker components fullfill this requirement.
 
-If Sawtooth ledger is used, then both Sawtooth (and the PDO transaction
-processors for Sawtooth) should be run on Ubuntu 16.04. If Microsoft CCF
-ledger is used, both CCF and the PDO transaction processor should be run
-on Ubuntu 18.04.
+If Microsoft CCF ledger is used, both CCF and the PDO transaction processor
+should be run on Ubuntu 18.04.
 We provide Docker images to run the ledger in the supported environment.
```

Comment:
this is not true. we should be running everything on ubuntu 20.04.

Reply:
It "was" true.

Reply:
Then change it?

Reply:
It's done